h2o-algos/src/test/java/hex/ensemble/StackedEnsembleTest.java

```diff
@@ -1529,4 +1529,137 @@ public void testMetalearnerTransformWorks() {
+            // Configure Stacked Ensemble
+            stackedEnsembleParameters._train = training_frame._key;
+            stackedEnsembleParameters._response_column = training_frame._names[idx];
+            stackedEnsembleParameters._metalearner_algorithm = Algorithm.glm;
+            stackedEnsembleParameters.initMetalearnerParams(Algorithm.glm);
+            GLMModel.GLMParameters glmParams =  (GLMModel.GLMParameters) stackedEnsembleParameters._metalearner_parameters;
+            glmParams._generate_scoring_history = true;
+            glmParams._score_iteration_interval = (glmParams._valid == null) ? 5 : -1;
+            glmParams._non_negative = true;
+            glmParams._alpha = new double[] {0.5, 1.0};
+            glmParams._standardize = false;
+
+            stackedEnsembleParameters._base_models = new Key[]{gbm._key, drf._key};
+            stackedEnsembleParameters._seed = seed;
+            stackedEnsembleParameters._score_training_samples = 0; // don't subsample dataset for training metrics so we don't randomly fail the test
+            // Invoke Stacked Ensemble and block till end
+            StackedEnsemble stackedEnsembleJob = new StackedEnsemble(stackedEnsembleParameters);
+            // Get the stacked ensemble
+            stackedEnsembleModel = stackedEnsembleJob.trainModel().get();
+
+            Frame training_clone = new Frame(training_frame);
+            DKV.put(training_clone);
+            Scope.track(training_clone);
+            preds = stackedEnsembleModel.score(training_clone);
+            final boolean predsTheSame = stackedEnsembleModel.testJavaScoring(training_clone, preds, 1e-15, 0.01);
+            Assert.assertTrue(predsTheSame);
+            Assert.assertTrue(stackedEnsembleJob.isStopped());
+
+            ModelMetrics training_metrics = stackedEnsembleModel._output._training_metrics;
+            ModelMetrics training_clone_metrics = ModelMetrics.getFromDKV(stackedEnsembleModel, training_clone);
+            Assert.assertEquals(training_metrics.mse(), training_clone_metrics.mse(), 1e-15);
+            training_clone.remove();
+
+
+
+        } finally {
+            if (training_frame != null) training_frame.remove();
+            if (gbm != null) {
+                gbm.delete();
+                for (Key k : gbm._output._cross_validation_predictions) k.remove();
+                gbm._output._cross_validation_holdout_predictions_frame_id.remove();
+                gbm.deleteCrossValidationModels();
+            }
+            if (drf != null) {
+                drf.delete();
+            }
+
+            if (preds != null) preds.delete();
+
+            Set<Frame> framesAfter = new HashSet<>(framesBefore);
+            framesAfter.removeAll(Arrays.asList(Frame.fetchAll()));
+
+            Assert.assertEquals("finish with the same number of Frames as we started: " + framesAfter, 0, framesAfter.size());
```

Comment:
is it part of this test? specific to this use case? Otherwise it looks like a smuggled test.

Reply:
smuggled test, I removed it now. Thanks!